.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
I think this can be misleading and needs another description; After all, an Enhancement also help us improve.

Reply:
@pheerai Well, this is the GiHub default :D Any better suggestions?

Reply:
```suggestion
about: Create a report to help us geting rid of errors
```